src/ZPublisher/Converters.py

```diff
@@ -135,7 +135,7 @@ def field2lines(v):
     if isinstance(v, (list, tuple)):
         result = []
         for item in v:
-            result.append(str(item))
+            result.append(field2bytes(item))
         return result
     return field2text(v).splitlines()
 
```

Comment:
Having the last return statement with `field2text` (which calls `field2string`) and the changed one using `field2bytes` looks inconsistent to me. Does the last line also have to be changed to use `field2bytes`?

Reply:
I agree, but I guess we should first sort out and agree which direction we want to take, i.e. lists of text vs lists of bytes.  There are no hard reasons for either I suppose, but like I said, we think "bytes" feel more right.

In any case, the `str` is wrong and cannot stay (imho).